acme.go

```diff
@@ -116,7 +117,8 @@ type AuthzID struct {
 // Client implements ACME spec.
 type Client struct {
 	http.Client
-	Key *rsa.PrivateKey
+	// Key.Public() must be *rsa.PublicKey
+	Key crypto.Signer
```

Comment:
Okay. Trying to *view* PRs on Gerrit is enormously painful already, so I'll keep this as an internal fork then. Thanks for your time.